app/src/main/res/layout/item_contact.xml

```diff
@@ -4,26 +4,37 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
+    xmlns:fresco="http://schemas.android.com/apk/res-auto"
     android:background="?android:attr/selectableItemBackground"
     android:paddingStart="@dimen/screen_edge_left_and_right_padding"
     android:paddingTop="@dimen/chat_item_top_and_bottom_padding"
     android:paddingEnd="@dimen/screen_edge_left_and_right_padding">
 
+
+    <com.facebook.drawee.view.SimpleDraweeView
+        android:id="@+id/contact_image_avatar"
+        android:layout_width="40dp"
+        android:layout_height="40dp"
+        android:background="@drawable/widechat_circle_shadow"
+        app:roundedCornerRadius="3dp"
+        fresco:actualImageScaleType="centerCrop"
+        fresco:roundAsCircle="true"
+        fresco:roundingBorderColor="@color/colorWhite"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        fresco:roundingBorderPadding="2dp"
+        fresco:roundingBorderWidth="2dp" />
+
+
     <TextView
-        android:id="@+id/contact_detail"
+        android:id="@+id/phone_number"
```

Comment:
I think that I was wrong in my initial comment about this......  I can now see that contact_detail is better than contact_phone, because this is a placeholder for either the phone number or the email, depending on which kind of contact it is.  Can we revert this to "contact_detail" please?